virtual_workstation_setup.sh

```diff
@@ -100,6 +101,28 @@ setup_cr() {
     popd
 }
 
+setup_jira_auth() {
+    # Get the user's JIRA username
+    read -p "JIRA username: " jira_username
```

Comment:
Suggest adding a bit more to the prompt here and maybe some kind of message on where to modify the name if they get it wrong. new users won't necessarily know whether the username is the full email.

Reply:
done